scripts/rook.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash -e
```

Comment:
why this is required?

Reply:
I just copied from minikube.sh and I think it means bash script. or i missed something?

Reply:
I mean we have rook.sh and deploy-rook.sh in the script folder, do we need both?

Reply:
oh, It was my mistake. I deleted it. Thanks!